projects/novo-examples/src/components/data-table/data-table-rows/data-table-rows-example.css

```diff
@@ -1 +1,8 @@
 /** No CSS for this example */
+.fixedWindowSize {
+  height: 600px;
+}
+
```

Comment:
the height is always going to be 600px?

Reply:
It's just for this demo example to create a scrollable window so you can test/see the sticky header in action.

Reply:
aha. sorry, Missed the example piece :)